components/ShoppingListEditor/InputDropDown.js

```diff
@@ -1,11 +1,24 @@
 import styled from "styled-components";
 
-export default function InputDropDown({ optionElements, ariaLabel, onAdd }) {
+export default function InputDropDown({
+  optionElements,
+  ariaLabel,
+  onAdd,
+  onReset,
+}) {
+  function handleClick(element) {
+    onAdd(element);
+    onReset();
```

Comment:
Why not combining both into one function? Or do you call one of these functions independently from another?